quarkus-integration/quarkus/deployment/src/test/java/ai/timefold/solver/quarkus/TimefoldProcessorSolverPropertiesTest.java

```diff
@@ -52,7 +55,7 @@ void solverProperties() {
         assertEquals(DomainAccessType.REFLECTION, solverConfig.getDomainAccessType());
         assertEquals(null,
                 solverConfig.getScoreDirectorFactoryConfig().getConstraintStreamImplType());
-
+        assertNotNull(solverConfig.getNearbyDistanceMeterClass());
```

Comment:
I obviously didn't notice before, but we use AssertJ, not JUnit assertions.

Reply:
Let me address that in this PR.